packages/wpcom-template-parts/src/universal-header-navigation/index.tsx

```diff
@@ -110,6 +110,57 @@ const UniversalNavbarHeader = ( {
 															type="dropdown"
 															target="_self"
 														/>
+														{ isEnglishLocale && (
+															<ClickableItem
+																titleValue={ __( 'Website Design Services', __i18n_text_domain__ ) }
+																content={ __( 'Website Design Services', __i18n_text_domain__ ) }
+																urlValue={ localizeUrl(
+																	'//wordpress.com/built-by/?ref=main-menu'
+																) }
+																type="dropdown"
+																target="_self"
+															/>
+														) }
```

Comment:
Question. I see 'Website Design Services' is gated by `isEnglishLocale` here. I assume that's because we only offer this service in English? Should we also be gating this in the WPCOM diffs?

Reply:
The WPCOM pages `/go`, `/learn` that I added were already only for English locales so I think it's safe to just add it 👍